File: 프로그래머스/unrated/181863. rny＿string/rny＿string.js
function solution(rny_string) {
    // 방법 2 - reaplce, join 함수
    return rny_string.split("m").join("rn")
    
    // // 방법 1 - replaceAll 함수
    // return rny_string.replaceAll("m", "rn")
}

File: 프로그래머스/unrated/181863. rny＿string/rny＿string.py
def solution(rny_string):
    # 방법 2 - replace 함수
    return rny_string.replace('m', 'rn')

    # # 방법 1 - split, join 함수 사용
    # return 'rn'.join(rny_string.split('m'))